explore/src/main/scala/explore/targeteditor/SearchForm.scala

```diff
@@ -70,6 +70,7 @@ object SearchForm {
         _ => // Auto-select name field on new targets.
           // Accessing dom elements by id is not ideal in React, but passing a ref to the <input> element of a
           // Form.Input is a bit convoluted. We should reevaluate when and if we switch to another component library.
+          // TODO: I don't think this even works.
```

Comment:
Unless I misunderstand what this `useEffectWithDepsBy` is supposed to do, it does not seem to be working in the current staging. I can try to fix it, or delete it, depending on whether we think it is necessary.

Reply:
Do we know if the element is even found? Seems like doing it via Ref would be better though the main question is:
Why are we doing this? to make it simpler to replace the currently selected text?
I may recall that detail and thinking it could be useful in many cases

Reply:
I don't think it is critical, though it maybe nice to have, if simple we could try to restore it but remove it if it is complicated

Reply:
I'm trying to debug it right now.

Reply:
Ah. Silly me. I didn't notice that it is only supposed to happen when you add a "New Empty Target". It is, indeed, functioning as intended.

Reply:
Yes that's what I recall, so you can type the new name right away